### PR TITLE
Add some repos

### PR DESCRIPTION
**Pull request by 0xSmit**
Added the following-
Plonky3 - Polygon's Plonky3 is a toolkit for implementing polynomial IOPs
Valida & supporting repos - Valida is Polygon's new stark based ZK VM
A few open source projects by Delta Blockchain Fund
Added Gravita protocol

### CODE REVIEW

File: data/ecosystems/g/gravita-protocol.toml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Ecosystem Level Information
+title = "Gravita Protocol"
```
Comment (eherrerosj) [inline data/ecosystems/g/gravita-protocol.toml:2]: could you please connect this ecosystem to Ethereum and Arbitrum?

File: data/ecosystems/p/polygon.toml [hunk 2]
```diff
@@ -5193,3 +5193,15 @@ url = "https://github.com/zmeghji/L2Token"
...
+url = "https://github.com/Plonky3/Plonky3"
+
+[[repo]]
+url = "https://github.com/valida-xyz/valida"
+
+[[repo]]
+url = "https://github.com/valida-xyz/valida-compiler"
+
+[[repo]]
+url = "https://github.com/valida-xyz/benchmarks"
```
Comment (eherrerosj) [inline data/ecosystems/p/polygon.toml:5207]: why should these repos by connected to Polygon? Would appreciate your helping validating these repos you added

**Review by eherrerosj — CHANGES_REQUESTED** [review]

File: data/ecosystems/g/gravita-protocol.toml (hunk 1 above)
Comment (0xSmit) [reply data/ecosystems/g/gravita-protocol.toml:2]: Do you mean to add these repos to the Ethereum and Arbitrum toml?

File: data/ecosystems/p/polygon.toml (hunk 2 above)
Comment (0xSmit) [reply data/ecosystems/p/polygon.toml:5207]: These repos are by the Polygon Zero team and will be used in conjunction with Polygon's 2.0 ZKEVM, read about polygon zero's involvement here in readme - https://github.com/Plonky3/Plonky3

File: data/ecosystems/g/gravita-protocol.toml (hunk 1 above)
Comment (0xSmit) [reply data/ecosystems/g/gravita-protocol.toml:2]: Hey @eherrerosj Have added the ecosystems

Comment (eherrerosj) [reply data/ecosystems/g/gravita-protocol.toml:2]: oh, actually Gravita should be a subecosystem of Ethereum and Arbitrum and not the other way around. In other words, the way you are requesting the changes is Gravita is a parent of Ethereum and Arbitrum in the taxonomy

File: data/ecosystems/p/polygon.toml (hunk 2 above)
Comment (eherrerosj) [reply data/ecosystems/p/polygon.toml:6312]: Ok, should we then just add them to the "Polygon Hermez" ecosystem toml file instead?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
Added a couple more comments

**Pull request merged**